File: Level1/6가운데글자가져오기/6.가운데글자가져오기.py
def solution(s):
    answer = ''
    stringLength = len(s)
    if stringLength%2==1:
        answer = s[int(stringLength/2)]
    else:
        start = int(stringLength/2)-1
        end = int(stringLength/2)
        answer = s[start:end+1]
    return answer

File: Level1/6가운데글자가져오기/6.가운데글자가져오기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    int stringLength = s.size();
    if(stringLength%2==1){
        answer = s[stringLength/2];
    }else{
        answer = s[(stringLength/2)-1];
        answer += s[stringLength/2];
    }
    return answer;
}